src/handlers/deltahandler.ts

```diff
@@ -39,7 +40,7 @@ export class DeltaHandler {
                                 sb.get(); // ?
```

Comment:
I know it's not part of your request but I'm wondering what the `?` means. Could we resolve it?